shared/actions/tracker.js

```diff
@@ -555,10 +555,17 @@ function _serverCallMap(
       addToIdleResponseQueue(() => {
         if (key.breaksTracking) {
           dispatch({type: Constants.updateEldestKidChanged, payload: {username}})
-          dispatch({
-            type: Constants.updateReason,
-            payload: {username, reason: `${username} has reset their account!`},
-          })
+          if (key.trackDiff.type === RPCTypes.IdentifyCommonTrackDiffType.newEldest) {
+            dispatch({
+              type: Constants.updateReason,
+              payload: {username, reason: `${username} has reset their account!`},
+            })
+          } else {
+            dispatch({
+              type: Constants.updateReason,
+              payload: {username, reason: `${username} has deleted a proof.`},
```

Comment:
I would change this to `deleted a PGP key`

Reply:
Copy that.